ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -47,33 +63,54 @@ function ScanConfigPage(): ReactElement {
         {
             name: PARAMETERS,
             id: PARAMETERS_ID,
-            component: <ScanConfigOptions formik={formik} />,
+            component: <ScanConfigOptions />,
         },
         {
             name: SELECT_CLUSTERS,
             id: SELECT_CLUSTERS_ID,
-            component: <ClusterSelection />,
+            component: (
+                <ClusterSelection clusters={clusters} isFetchingClusters={isFetchingClusters} />
+            ),
+            canJumpTo: Object.keys(formik.errors?.parameters || {}).length === 0,
+        },
+        {
+            name: SELECT_PROFILES,
+            id: SELECT_PROFILES_ID,
+            component: (
+                <ProfileSelection
+                    profiles={profiles || []}
+                    isFetchingProfiles={isFetchingProfiles}
+                />
+            ),
+            canJumpTo: Object.keys(formik.errors?.parameters || {}).length === 0,
+        },
+        {
+            name: REVIEW_CONFIG,
+            id: REVIEW_CONFIG_ID,
+            component: <ReviewConfig />,
```

Comment:
Does the last step need `nextButtonText` property for something like **Save** instead of **Next**?

Reply:
Good question, thats coming in PR for setting up the review component